lib/Core/Executor.cpp

```diff
@@ -1321,26 +1313,42 @@ Executor::toConstant(ExecutionState &state,
   if (ConstantExpr *CE = dyn_cast<ConstantExpr>(e))
     return CE;
 
-  ref<ConstantExpr> value;
-  bool success =
-      solver->getValue(state.constraints, e, value, state.queryMetaData);
-  assert(success && "FIXME: Unhandled solver failure");
-  (void) success;
+  ref<Expr> value;
+  if (auto found = seedMap.find(&state); found != seedMap.end())
+    value = getValueFromSeeds(found->second, e);
+  /* If no seed evaluation results in a constant, call the solver */
```

Comment:
Is there are reason for using C-style comments? It makes it harder to comment out blocks (due to nesting) and is not recommended by the LLVM coding standard:

https://llvm.org/docs/CodingStandards.html#comment-formatting

Reply:
No particular reason, and I think we can live with both.  I guess I could change it, but I'm not convinced it matters. And looking at the code, we seem to use both types of comments.  
To comment out code, I recommend `#if 0` / `#endif` which should work with both types of comments.

Reply:
> To comment out code, I recommend #if 0 / #endif which should work with both types of comments.

Common IDEs only support the traditional line/block comment via shortcuts but ok.